packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
-    }
-  }
-});
\ No newline at end of file
+import * as esJscore from 'https://esm.run/@es-js/core'
+import * as esJseshtml from 'https://esm.run/@es-js/eshtml'
+window.addEventListener('load', function () {
+  let etiquetas = document.getElementsByTagName('script')
+  for (var i = 0; i < etiquetas.length; i++) {
+    if (
+      etiquetas[i].hasAttribute('type') &&
+      (etiquetas[i].getAttribute('type') == 'text/esjs' ||
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)